doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -138,6 +138,16 @@ each replica, including the times at which it was created and dropped
 | `dropped_at`          | [`timestamp with time zone`] | The time at which the replica was dropped, or `NULL` if it still exists.                                                                  |
 | `credits_per_hour`    | [`numeric`]                  | The number of compute credits consumed per hour. Corresponds to [`mz_cluster_replica_sizes.credits_per_hour`](#mz_cluster_replica_sizes). |
 
+### `mz_managed_cluster_replicas`
+
+The `mz_managed_cluster_replicas` table lists the replicas that are part of the managed configuration of
+a managed cluster.
+
+<!-- RELATION_SPEC mz_internal.mz_managed_cluster_replicas -->
+| Field      | Type     | Meaning                                       |
+|------------|----------|-----------------------------------------------|
+| id         | [`text`] | The ID of a cluster replica.                  |
```

Comment:
In other tables we are (inconsistently) adding "Corresponds to ..." hints at the end. Might be worth doing that here too for `mz_cluster_replicas`.